lib/project_types/script/ui/error_handler.rb

```diff
@@ -22,6 +24,14 @@ def self.pretty_print_and_raise(e, failed_op: nil)
         display_and_raise(failed_op: failed_op, **messages)
       end
 
+      private_class_method def self.append_msg(full_msg, msg_to_append)
+        full_msg << if /\s$/.match?(full_msg)
+          msg_to_append
+        else
+          " #{msg_to_append}"
+        end
```

Comment:
Since we're always appending `msg_to_append`, we could remove it from the conditional and keep that focused on the space, i.e.
```ruby
full_msg << " " unless /\s$/.match?(full_msg)
full_msg << msg_to_append
```